src/dcqc/file.py

```diff
@@ -109,6 +109,7 @@ def get_file_type(cls, file_type: str) -> FileType:
 FileType("OME-TIFF", (".ome.tif", ".ome.tiff"), "format_3727")
 FileType("TSV", (".tsv"), "format_3475")
 FileType("BAM", (".bam"), "format_2572")
+FileType("FASTQ", (".fastq", ".fastq.gz"), "format_1930")
```

Comment:
@BWMac Hold this for a second, but I think we have `fq` and `fq.gz` as well.

Reply:
Looks like [BioPortal](https://bioportal.bioontology.org/ontologies/EDAM?p=classes&conceptid=format_1930) treats both extensions (`fq` and `fastq`) the same format too, so we could just add  `fq`and `fq.gz` to the tuple here.

Reply:
added and tested successfully.